deploy/builder/templates-install-bundle/clickhouse-operator-install-yaml-template-01-section-crd-01-chi-chit.yaml

```diff
@@ -308,6 +313,38 @@ spec:
                   enum:
                     - ""
                     - "RollingUpdate"
+                suspend: &TypeStringBool
```

Comment:
This is duplicate of the definition of the new type. First definition is made for 
```yaml
                stop: &TypeStringBool
                  type: string
```
please change to usage of the type, such as for example for
```yaml
                troubleshoot:
                  <<: *TypeStringBool
                  description: |
                    Allows to troubleshoot Pods during CrashLoopBack state.
```

Reply:
ideally you use something like kubebuilder to generate these things

Example: https://github.com/kubernetes-sigs/mcs-api/blob/master/pkg/apis/v1alpha1/serviceexport.go#L24